runtime-builder/Dockerfile

```diff
@@ -41,6 +41,7 @@ RUN curl -sSL https://storage.googleapis.com/golang/go${GO_VERSION}.linux-amd64.
 
 COPY go-build.sh /usr/local/bin/
 COPY go-run.sh /usr/local/bin/
+RUN chmod 755 /usr/local/bin/go-build.sh /usr/local/bin/go-run.sh
```

Comment:
I think so. Also I found this out by using my different machines where those files have different permission.